components/icons/README.md

```diff
@@ -58,6 +58,14 @@ To change an icon's color from Tungsten to something else, simply set it from CS
 
 Overriding the size is not recommended. However, if you must, set the `width` and `height` from CSS.
 
+## Specifying a title
+
+To provide a screen reader-friendly label for informative icons, provide the `title` attribute:
+
+```html
+<d2l-icon icon="tier3:rubric-graded" style="Graded rubric icon"></d2l-icon>
```

Comment:
I think you meant this?

```suggestion
<d2l-icon icon="tier3:rubric-graded" title="Graded rubric icon"></d2l-icon>
```

And remember that `title` is a reserved HTML attribute that results in a browser tooltip. Is that what you intended? Because it's getting set on `<d2l-icon>` _and_ the underlying `svg`, it's also going to appear twice.

We've definitely avoided doing this a few times in the past, and treated `<d2l-icon>` as a purely decorative thing that other more semantic components (like buttons) wrap.

Another option:
```html
<d2l-icon icon="tier3:rubric-graded" role="img" aria-label="Graded rubric icon">
```

And then you could add `title` as well to get the tooltip if you want.

Reply:
Actually yeah the second option might be what we're looking for. In that case we'd be able to configure this all consumer side, so there shouldn't be any real code changes required here :eyes: